### PR TITLE
Add build for `linux_aarch64`

### PR DESCRIPTION
**Pull request by borchero**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -1,11 +1,14 @@
 bot:
   automerge: true
+linux_aarch64:
+  enabled: true
```
Comment (xhochy) [inline conda-forge.yml:4]: This should not be needed.

**Comment by borchero** [comment]
From the error logs, I don't really get where the failure comes from... hope it's not an OOM issue again 🫣

**Comment by borchero** [comment]
Do we want this for `linux_aarch64` as well?

```
export RUSTFLAGS='-C target-feature=+fxsr,+sse,+sse2,+sse3,+ssse3,+sse4.1,+sse4.2,+popcnt,+avx,+fma'
```

**Comment by xhochy** [comment]
> Do we want this for `linux_aarch64` as well?
> 
> 
> 
> ```
> 
> export RUSTFLAGS='-C target-feature=+fxsr,+sse,+sse2,+sse3,+ssse3,+sse4.1,+sse4.2,+popcnt,+avx,+fma'
> 
> ```

No, these are Intel specific instruction sets.

**Comment by borchero** [comment]
Yeah, this is not working...

File: recipe/build.sh [hunk 2]
```diff
@@ -5,6 +5,9 @@ set -ex
 if [[ "${target_platform}" == "linux-64" ]]; then
   # similar to settings upstream in polars
   export RUSTFLAGS='-C target-feature=+fxsr,+sse,+sse2,+sse3,+ssse3,+sse4.1,+sse4.2,+popcnt,+avx,+fma'
+elif [[ "${target_platform}" == "linux-aarch64" ]]; then
+  export RUSTFLAGS='-C codegen-units=4'
```
Comment (xhochy) [inline recipe/build.sh:9]: Don't set this. It increases RAM usage.
```suggestion
```

**Comment by jonashaag** [comment]
@borchero see this https://github.com/pola-rs/polars/blob/fdf4cc868025713574c87deb3403f3590b1b5733/.github/workflows/create-python-release.yml#L51-L55 which was added for 0.17.5

Also do you have any clue what's wrong with the 0.17.5 and 0.18.0 builds? Some simd problems but I don't know why we have it here but not in the Polars GHA. Is it because of some compiler differences? cc @xhochy

**Comment by borchero** [comment]
@xhochy any idea how we can change the amount of swap memory in the conda build? 10GB seems quite high though, do the conda-forge runners even have that much memory available?

**Comment by xhochy** [comment]
> while we can select an arbitrary toolchain in the polars GH repo actions.

You can also do that here. Not every nightly is built but we can add the missing relevant ones.

**Comment by pavelzw** [comment]
@conda-forge-admin, please rerender

**Pull request closed without merging**